settings.gradle.kts

```diff
@@ -14,7 +14,6 @@ dependencyResolutionManagement {
         mavenLocal()
         mavenCentral()
         google()
-        maven(url = "https://androidx.dev/storage/compose-compiler/repository/")
```

Comment:
This line also can be removed because it was just required to use pre-release version of compose compiler.